### ISSUE TITLE
Botsing Behavioral model seeding seems not execute test cases in Atos UC Supersede IF

### ISSUE THREAD

**Issue by jesus-gorronogoitia** [STAMP-project/botsing#90]
#### Characteristics
- **Issue Type**: Test report
- **Reproducibility**: Always
- **Severity**: Minor
- **Tool/Service/Component**: Botsing behavioral model seeding, 1.0.7
- **Execution Environment**: Linux CentOS 7
- **Reporter**: Jesús Gorroñogoitia <jesus.gorronogoitia@atos.net>

#### Description
According to @xdevroey Botsing behavioral model generation conducts an static and dynamic analysis of the SUT test suites. Therefore, the dynamic analysis actually executes the SUT test cases. However, when I run Botsing behavioral model generation it seems that only static analysis is executed, because: i) I don't see test execution logs, ii) the overall behavioral model generatation takes less than a minute, while the SUT test execution takes several minutes.

Is this correct? Should dynamic model generation be executed by default or is it disabled?

#### Steps to reproduce
I am executing Botsing and behavioral model generation within a Docker container that hosts the Botsing libraries and some scripts I've created, plus the SUT (Supersede IF) libraries, source code and compiled sources.
First I compute the classpath using this command:
```
#Compile IF with Maven
mvn clean package -DskipTests
#Copy dependencies
mvn org.apache.maven.plugins:maven-dependency-plugin:copy-dependencies
#Compute IF classpath
for i in ./dependency/*.jar; do echo -n $i":"; done >> results/classpath.txt
```
Then, I generate the behavioral model with this script:
`java -d64 -Xmx10000m -jar supersede-if-botsing/botsing-model-generation-1.0.7.jar -project_cp "$(cat results/classpath.txt)" -project_prefix "eu.supersede.integration" -out_dir "results/models"
`
Total model generation execution takes less than a minute.
Generate models can be browse in IF GitHub repository: [Botsing behavioral model](https://github.com/supersede-project/integration/tree/stamp-baseline/IF/API/eu.supersede.if.api/Botsing/supersede-if-botsing/results/models)

#### [Help on issue template](https://github.com/STAMP-project/EvoCrash/blob/master/.github/ISSUE_DOC.md) 
Preview to follow the link or open file .github/ISSUE_DOC.md

**Comment by pderakhshanfar** [comment]
@jesus-gorronogoitia,
Yes, dynamic analysis is enabled by default. In your case, you can see [here](https://github.com/supersede-project/integration/blob/stamp-baseline/IF/API/eu.supersede.if.api/Botsing/supersede-if-botsing/results/models/carvedTests/tests.xml) that there is one carved test suite (`eu.supersede.integration.api.pubsub.test.EvolutionAlertMessageListenerTest`). However, it seems that the carver could not execute the other test suites. Can you please attach the execution log of the model generator?

**Comment by jesus-gorronogoitia** [comment]
Hi @pderakhshanfar See model generator logs [here](https://github.com/supersede-project/integration/blob/stamp-baseline/IF/API/eu.supersede.if.api/Botsing/supersede-if-botsing/results/generate_behavioral_models.log.tgz)

**Comment by pderakhshanfar** [comment]
The carving process (in EvoSuite) was not successful in executing some test cases. But, there is no error message from EvoSuite. In my opinion, it does not execute because of either security issues or execution timeout. For the latter reason, you can increase the carving timeout by `-Dcarving_timeout=<time_in_seconds>`. For security issues, we need to debug it.

Additionally, I see from the log that it is generating some models. You can also see if the existing model can help the search process or not.
